profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/variables.tf.ctmpl

```diff
@@ -4,6 +4,11 @@ provider "google-beta" {
   region = "us-central1"
 }
 
+variable "vault_path_prefix" {
+  default = "{{env "VAULT_PATH_PREFIX"}}"
+  description = "The prefix path at which to store vault values"
+}
+
```

Comment:
Don't need this if we're hard-coding the Vault path and it doesn't change between envs